doc/release/1.16.0-notes.rst

```diff
@@ -280,6 +280,14 @@ and other path-like objects in addition to a file object. Furthermore, the
 ``np.load`` function now also supports path-like objects when
 using memory mapping (``mmap_mode`` keyword argument).
 
+``ndrange`` a multi-dimensional range-like object
+-------------------------------------------------
+``ndrange`` behaves much like the Python 3 ``range`` object and allows
+multi-dimensional slicing, iteration, reversal and efficient containment lookup.
```

Comment:
oh also, I think the correct word is "membership", not "containment", to follow python docs.